heapster-saw/examples/_CoqProject

```diff
@@ -6,11 +6,11 @@
 linked_list_gen.v
 linked_list_proofs.v
 xor_swap_gen.v
-#xor_swap_proofs.v
+xor_swap_proofs.v
 xor_swap_rust_gen.v
 #xor_swap_rust_proofs.v
 c_data_gen.v
-#c_data_proofs.v
+c_data_proofs.v
```

Comment:
I'm surprised that this isn't adding back more of the `_proofs.v` files that you've changed in this PR, such as `arrays_proofs.v`. Any reason not to?

Reply:
I haven't finished the proofs in `arrays_proofs.v`, I only fixed the first one which is the one mentioned in the tutorial.